sklearn/ensemble/voting.py

```diff
@@ -264,6 +267,7 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : object
         """
+        check_classification_targets(y)
```

Comment:
is this necessary ?
Wouldn't it be caught by anyone of the `estimators` ?

Reply:
Yes, this is necessary. The label encoder will succeed to transform continuous variable into classification labels while it should fail as done in the `check_classification_targets`.